### PR TITLE
Added Median for background estimation

### PR DESCRIPTION
**Pull request by siddharthlal25**
Added median statistic for background estimation as mentioned in issue #12

### CODE REVIEW

**Comment by giordano** [comment]
This PR seems to be based on #14?  Could you please start a branch out of `master`?

**Comment by siddharthlal25** [comment]
This PR uses the function `sigma_clip` which is implemented in PR #14 so I have based it on `sigma` branch.

**Comment by mileslucas** [comment]
Right. This makes managing PRs much harder for us. Now we have to keep in mind that we can't merge this without merging the sigma-clip method. PRs should usually be based on the current upstream master.

**Comment by siddharthlal25** [comment]
Understood, I will remove the instances of sigma clipping from the median part and rebase it on the current master branch. :slightly_smiling_face:

File: src/Background/stat_estimators.jl [hunk 1]
```diff
@@ -20,3 +20,24 @@ julia> estimate_background(Mean, data, dims=1)
 struct Mean <: BackgroundEstimator end
 
 estimate_background(::Mean, data; dims = :) = mean(data, dims = dims)
+
+"""
+    Median <: BackgroundEstimator
+
+This estimator returns the median of the input.
+
+#Example
```
Comment (mileslucas) [inline src/Background/stat_estimators.jl]: Needs to be `# Example`

**Pull request merged**